packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-installation/illustrations-installation.component.html

```diff
@@ -0,0 +1,74 @@
+<elvia-segmented-control
+  type="text"
+  size="large"
+  [value]="activeTabIndex"
+  [items]="tabs | tabToSegmentedControlItem"
+  (valueOnChange)="setActiveTab($any($event).detail.value)"
+/>
+
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <strong>Install package</strong>
+    <div class="e-mt-8">
+      <app-copy message="npm install @elvia/elvis-illustrations">
+        <code>npm install &#64;elvia/elvis-illustrations</code>
+      </app-copy>
+    </div>
+    <div class="e-text-md e-mt-8">or</div>
+    <div class="e-mt-8">
+      <app-copy message="yarn add @elvia/elvis-illustrations">
+        <code>yarn add &#64;elvia/elvis-illustrations</code>
+      </app-copy>
+    </div>
+  </li>
+  <li>
+    <strong>Import component</strong>
+    <div class="e-mt-8">
+      <app-copy message="import '@elvia/elvis-illustrations/ILLUSTRATION-NAME';">
+        <code>import '&#64;elvia/elvis-illustrations/ILLUSTRATION-NAME';</code>
+      </app-copy>
+      <div class="e-text-md e-mt-8">
+        <strong>NB!</strong> The component should only be imported once, e.g. in the main.ts file.
```

Comment:
Bør `main.ts` stå i monospace?

Reply:
It's now the same way as it is in the normal installation guide, so idk, maybe? 🤷